### PR TITLE
Add PR checklist, reorganise images

### PR DESCRIPTION
**Pull request by kaustubhhiware**
### Description

* Added mandatory PR checklist in Contributing.md
* Reorganised images folder

### Type of Change:
**Delete irrelevant options.**

- Quality Assurance

### How Has This Been Tested?
N/A

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by kaustubhhiware** [comment]
@xprilion @techytushar @roopalJazz @farhaanbukhsh please have a look, need to merge before other participants can send PRs.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Describe the tests you ran to verify your changes. Provide instructions or GIFs
 ### Checklist:
 **Delete irrelevant options.**
 
+- [ ] I have ensured my PR satisfies the checklist mentioned in CONTRIBUTING.md.
```
Comment (techytushar) [inline PULL_REQUEST_TEMPLATE.md]: we can have a link to that section here, for easy access

**Comment by kaustubhhiware** [comment]
@farhaanbukhsh please suggest which file we should add it to, I'll do that immediately.

**Comment by farhaanbukhsh** [comment]
@kaustubhhiware In the checklist file you can add this.

**Comment by kaustubhhiware** [comment]
@farhaanbukhsh have a look.

**Pull request merged**